README.md

```diff
@@ -34,7 +44,7 @@ of R can be found here: https://cdn.rstudio.com/r/versions.json
 R_VERSION=3.5.3
 ```
 
-### Download and install R
+### Download and install R Manually
```

Comment:
The "Specify R version" section above this is also part of the manual install process. What if we wrote something like this at the bottom of the Quick Installer section: "Or follow the steps below to install R manually". Or we could split the Installation section into Quick Installer and Manual Installation subsections.